templates/definition/tariff/octopus-productcode.yaml

```
@@ -20,7 +20,15 @@ params:
     help:
       de: "Die DNO-Region, in der Sie sich befinden. Weitere Informationen: https://www.energy-stats.uk/dno-region-codes-explained/"
       en: "The DNO region you are located in. More information: https://www.energy-stats.uk/dno-region-codes-explained/"
+  - name: directDebit
```

```suggestion
  - name: directdebit
```

we don't usually use camel case in templates imho. It's unfortuante we've changed this in this template.

---

hmm, considering #20304 was caused by inappropriate camel casing, maybe a stronger deprecation or explicit handling of both options is in order?

---

Possibly. The rest of the template options use it so I followed suit with this one - I can add a commit that switches all the options to lower case?

---

Took another look. Leave as is. We'll survive...

---

Likely one to consider as a wider unpicking of camelCasing in templates / config. 👍